code/stable/gamephys/SRS/Chipmunk_SRS.tex

```diff
@@ -138,6 +140,10 @@ \subsection{Table of Symbols}
 \\
 ${\mathbf{v}^{BP}}$ & Velocity Of the Point of Collision P in Body B & $\frac{\text{m}}{\text{s}}$
 \\
+${\mathbf{v}_{1}}$ & Velocity Of the Firstbody & $\frac{\text{m}}{\text{s}}$
```

Comment:
Something is weird "Firstbody" - there is a space missing.

Reply:
fixed